example/android/javalib/1-hello-world/app/src/test/java/com/helloworld/app/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.helloworld.app;
+
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+/**
+ * Example local unit test, which will execute on the development machine (host).
+ *
+ * @see <a href="http://d.android.com/tools/testing">Testing documentation</a>
+ */
+public class ExampleUnitTest {
+  @Test
+  public void addition_isCorrect() {
+    assertEquals(4, 2 + 2);
```

Comment:
Can we make this unit test depend on the code in the `src/main/` folder? So we can ensure that the classpaths are properly wired up

Reply:
Same for the kotlin example below